configure.ac

```diff
@@ -179,6 +179,17 @@ AC_CHECK_LIB(tpms,
 )
 AC_SUBST([LIBTPMS_LIBS])
 
+PKG_CHECK_EXISTS([libtpms >= 0.9], [
```

Comment:
If you drop patch 4 this one is probably not necessary, either.

Reply:
right